torch/testing/_internal/common_methods_invocations.py

```diff
@@ -219,7 +224,82 @@ def sample_inputs(self, device, dtype, requires_grad=False):
                                         low=low, high=high,
                                         requires_grad=requires_grad)),)
 
+class SvdInfo(OpInfo):
+    """Operator information for 'torch.svd'
+    """
+
+    def sample_inputs(self, device, dtype, requires_grad=False):
```

Comment:
https://github.com/pytorch/pytorch/pull/48627 will provide a way to provide a custom sample_inputs that doesn't require defining a new class, which should simplify this code.